pkg/roachprod/testselector/main.go

```diff
@@ -0,0 +1,59 @@
+// Copyright 2024 The Cockroach Authors.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+// reference - https://cockroachlabs.atlassian.net/wiki/spaces/~7120207825326fb5e546c194029506f2c5335e/pages/3596091522/Test+Selection+Criteria
+
+package main
```

Comment:
Bit odd that this is not a library, right? Why is this a standalone binary? To use it from `roachprod`, you would have to build this binary and expose it separately. Instead it should be a library that you call into.

Reply:
Oh, I see. This uploads a CSV that `roachprod` will download later.

Reply:
So presumably you are planning on setting up a TC job that executes this binary.

Reply:
yes